shared/src/commonMain/kotlin/com/kcc/kmmhackathon/shared/entity/Entity.kt

```diff
@@ -23,4 +23,17 @@ data class Meal(
     val locationLat: Float,
     @SerialName("locationLong")
     val locationLong: Float
+)
+
+@Serializable
+data class Quantity(
+    @SerialName("quantity")
+    val quantity: Int
+)
```

Comment:
Added this in so that I can send just the quantity as the body for a patch request.